nix-darwin/RewriteLibs.hs

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env nix-shell
```

Comment:
This module has a shebang but no main function. Maybe set this up as a `rewrite-libs` program with main and replace call to `patch-prepare` with `rewrite-libs --out-dir $out/bin $out/bin/*`